docs/upgrade/v1-4-0-to-v1-4-1.md

```diff
@@ -80,4 +80,27 @@ A possible cause is processes related to orphan engines of the Longhorn Instance
 
 Related issues:
   - [[BUG] v1.4.0 -> v1.4.1-rc1 upgrade stuck in Pre-drained and the node stay in Cordoned](https://github.com/harvester/harvester/issues/7366)
-  - [[IMPROVEMENT] Cleanup orphaned volume runtime resources if the resources already deleted](https://github.com/longhorn/longhorn/issues/6764)
\ No newline at end of file
+  - [[IMPROVEMENT] Cleanup orphaned volume runtime resources if the resources already deleted](https://github.com/longhorn/longhorn/issues/6764)
+
+### 2. Upgrade Harvester UI Extension to 1.0.3 on Rancher v2.10.1
+
+To import Harvester v1.4.1 on Rancher v2.10.1, harvester-ui-extension requires upgrade to version `1.0.3`.
```

Comment:
Do we need to clarify that Harvester v1.0.3 only works with Rancher v2.10.1 or higher?